lib/vsc/install/shared_setup.py

```diff
@@ -165,6 +165,7 @@ def _log(self, level, msg, args):
 SHEBANG_ENV_PYTHON = '#!/usr/bin/env python'
 SHEBANG_NOENV_PYTHON = '#!/usr/bin/python-noenv'
 SHEBANG_PYTHON_E = '#!/usr/bin/python -E'
+SHEBANG_STRIPPED_ENV_PYTHON = '#!/usr/bin/python-stripped-env'
```

Comment:
don't forget to change   line 609:             ` code = pyshebang_reg.sub(SHEBANG_PYTHON_E, code)` to `code = pyshebang_reg.sub(SHEBANG_STRIPPED_ENV_PYTHON, code)`